[2022-08-11 23:44:49,717] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ETL_Raízen.creating_raw_data_dir manual__2022-08-11T23:44:45.811252+00:00 [queued]>
[2022-08-11 23:44:49,733] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ETL_Raízen.creating_raw_data_dir manual__2022-08-11T23:44:45.811252+00:00 [queued]>
[2022-08-11 23:44:49,734] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-08-11 23:44:49,735] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-08-11 23:44:49,735] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-08-11 23:44:49,769] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): creating_raw_data_dir> on 2022-08-11 23:44:45.811252+00:00
[2022-08-11 23:44:49,778] {standard_task_runner.py:52} INFO - Started process 6849 to run task
[2022-08-11 23:44:49,784] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_Raízen', 'creating_raw_data_dir', 'manual__2022-08-11T23:44:45.811252+00:00', '--job-id', '95', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Raizen.py', '--cfg-path', '/tmp/tmpsfx9h72x', '--error-file', '/tmp/tmpwg_1d7al']
[2022-08-11 23:44:49,786] {standard_task_runner.py:80} INFO - Job 95: Subtask creating_raw_data_dir
[2022-08-11 23:44:49,921] {task_command.py:369} INFO - Running <TaskInstance: ETL_Raízen.creating_raw_data_dir manual__2022-08-11T23:44:45.811252+00:00 [running]> on host 53e7d4a502e6
[2022-08-11 23:44:50,185] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ETL_Raízen
AIRFLOW_CTX_TASK_ID=creating_raw_data_dir
AIRFLOW_CTX_EXECUTION_DATE=2022-08-11T23:44:45.811252+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-11T23:44:45.811252+00:00
[2022-08-11 23:44:50,189] {ETL_Raizen.py:28} INFO - Dir already exists
[2022-08-11 23:44:50,190] {python.py:173} INFO - Done. Returned value was: None
[2022-08-11 23:44:50,253] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ETL_Raízen, task_id=creating_raw_data_dir, execution_date=20220811T234445, start_date=20220811T234449, end_date=20220811T234450
[2022-08-11 23:44:50,320] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-11 23:44:50,460] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
